cs130_front/src/components/Listings/ListingItem.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import Text from '../Text/Text';
+import * as Fonts from '../../constants/Fonts';
+import * as Colors from '../../constants/Colors';
+import './ListingItem.css';
+
+export default function ListingItem(props) {
+    return(
+        <div className="box-shadow" style={{textAlign: "center"}}>
+            <div style={{display: "flex", flexDirection: "column"}}>
+                <div className="row">
```

Comment:
You can add an onClick to this div specifically to go to poster's profile. Might want to make this a "clickable row" or something because on hover the pointer should change.

Reply:
Done